hintman.cabal

```diff
@@ -24,9 +24,14 @@ library
   hs-source-dirs:      src
   exposed-modules:     Hintman
                        Prelude
+                       Server
 
   build-depends:       base-noprelude
                      , relude
+                     , servant
```

Comment:
Since `servant-generic` was added to servant in `0.14.1`, would it make sense to add a `>=` constrant here and below?

Reply:
Agree